scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -1564,7 +1578,8 @@ static __noinline int consume_non_open(struct cost *costc, s32 cpu, u32 my_llc_i
 					return -EINVAL;
 				}
 				layer = MEMBER_VPTR(layers, [layer_idx]);
-				if (has_budget(costc, layer) == 0)
+				if (layer->kind == LAYER_KIND_OPEN ||
```

Comment:
Is this related to the main change of the diff?

Reply:
I can pull this into a separate diff, good call.